build/Targets/GenerateInternalsVisibleTo.targets

```diff
@@ -34,6 +34,9 @@
     <InternalsVisibleToRazor>
       <Visible>false</Visible>
     </InternalsVisibleToRazor>
+    <InternalsVisibleToRemoteLS>
```

Comment:
❗️ I believe this needs a `CreateItem` down below (~line 112)

Reply:
thanks! Added this & a GenerateInternalsVisibleToFile condition to match the others.